orchagent/portsorch.cpp

```diff
@@ -1463,6 +1517,12 @@ bool PortsOrch::addBridgePort(Port &port)
         return false;
     }
 
+    if (!setHostIntfsStripTag(port, SAI_HOSTIF_VLAN_TAG_KEEP))
+    {
+        SWSS_LOG_ERROR("Failed to set %s for hostif of port %s",
+                hostif_vlan_tag[SAI_HOSTIF_VLAN_TAG_KEEP], port.m_alias.c_str());
+        return false;
+    }
```

Comment:
let's move this to addVlanMember(), in the untagged mode, we do not need to call KEEP.